test/index.js

```diff
@@ -802,7 +802,7 @@ test ('lte', function() {
   eq (Z.lte (Identity (Identity (Identity (0))), Identity (Identity (Identity (0)))), true);
   eq (Z.lte (Identity (Identity (Identity (0))), Identity (Identity (Identity (1)))), true);
   eq (Z.lte (Identity (Identity (Identity (1))), Identity (Identity (Identity (0)))), false);
-  eq (Z.lte (Lazy$of (0), Lazy$of (0)), false);
+  eq (Z.lte (Lazy$of (0), Lazy$of (0)), true);
```

Comment:
I believe this started returning `true` because there is no `Function$prototype$ord` implementation, so two functions will be structurally compared: `Z.lte (() => {}, () => {})` now evaluates to `true` because the two values are structurally "less than or equal".

Oddly, in #154, `Z.equals (() => {}, () => {})` evaluates to `false`, because we have an implementation of Setoid for functions that uses reference equality.

Reply:
I would expect that the following always holds: `Z.equals (a, b) && Z.lte (a, b) == Z.equals (a, b)`, but this rule is now broken.